examples/07-periodic_hills/periodic_hills.prm

```diff
@@ -44,7 +44,18 @@ subsection flow control
     set volumetric flow rate       = -6.41025
     set boundary id    		   = 0
     set flow direction 		   = 0
-    set initial beta		   = 0.5
+    set initial beta		   = 5.3764
+end
+
+#---------------------------------------------------
+# Post-Processing
+#---------------------------------------------------
+subsection post-processing
+    set calculate average velocities    = true
+    set initial time 		        = 0.1
+    set boundary id    	           	= 0
+    set flow direction 		        = 0
+    set nondimensionalization           = true
```

Comment:
Is it really necessary to do the dimensionless analysis from the code directly? What prevents doing it from outside directly into paraview?

Reply:
Ok I will remove it, I didn't know we could do it in Paraview.